docs/index.rst

```diff
@@ -121,14 +121,22 @@ You may want to configure the ``LEVEL`` of the Polaris logger differently depend
 Environment Variables
 ^^^^^^^^^^^^^^^^^^^^^
 
-Polaris uses environment variables that should be defined in the
-environment or included in ``BASE_DIR/.env`` or ``POLARIS_ENV_PATH``.
+Polaris uses environment variables that should be defined in the environment or included in ``BASE_DIR/.env`` or ``POLARIS_ENV_PATH``.
 ::
 
     STELLAR_NETWORK_PASSPHRASE="Test SDF Network ; September 2015"
     HORIZON_URI="https://horizon-testnet.stellar.org/"
     HOST_URL="https://example.com"
 
+Polaris also supports specifying your environment variables in your project's ``settings.py``. However, any variable Polaris expects in the environment must be prepended with ``POLARIS_`` if declared in `settings.py``. For example,
+::
+
+    POLARIS_STELLAR_NETWORK_PASSPHRASE = "Test SDF Network ; September 2015"
+    POLARIS_HORIZON_URI = "https://horizon-testnet.stellar.org/"
+    POLARIS_HOST_URL = "https://example.com"
+
+If a variable cannot be found in the environment, Polaris will look in the ``settings.py`` file. Be careful not to check in secrets like your ``SERVER_JWT_KEY`` into your version control history, especially if your repository is not private.
```

Comment:
@JakeUrban This makes it sound like Polaris will look in the `settings.py` file directly. I suggest mentioning that Polaris will look for settings in `django.conf` and `settings.py` is where these settings are usually defined, but also mention that `local_settings.py` should be used to store secret settings:
```
# bottom of settings.py
try:
    from local_settings import *
except ImportError:
    pass
```

Reply:
I can mention that Polaris reads specifically from `django.conf` instead of an actual `settings.py` file, but I think the `local_settings` module you use doesn't need to be mentioned in the docs. 

Keeping python files secret is a bit unusual, as opposed to secret environment variables. If Polaris users are declaring all their variables in python, they're likely declaring them explicitly in `settings.py` and keeping the entire repository secret/private. Even if they use your approach, its implied that other modules can be imported into the main `settings.py`.